plugins/modules/vmware_guest_cross_vc_clone.py

```diff
@@ -302,6 +302,13 @@ def populate_specs(self):
         # populate service locator
         self.service_locator.instanceUuid = self.destination_content.about.instanceUuid
         self.service_locator.url = "https://" + self.destination_vcenter + ":" + str(self.params['port']) + "/sdk"
+        # If ssl verify is false, we ignore it also in the clone task by fetching thumbprint automatically
+        if not self.destination_vcenter_validate_certs:
+            self.service_locator.sslThumbprint = self.get_cert_fingerprint(
+                self.destination_vcenter,
+                self.params['port'],
```

Comment:
```suggestion
                self.destination_vcenter_port,
```

When getting the certificate of the destination vCenter, we should use its port. `self.params['port']` is the port of the _source_ vCenter. I've never seen a vCenter not running on 443, but you never know :shrug: